client/landing/stepper/declarative-flow/internals/hooks/use-launchpad-decider/index.tsx

```diff
@@ -1,12 +1,13 @@
 import { updateLaunchpadSettings } from '@automattic/data-stores';
 import { addQueryArgs } from '@wordpress/url';
 import { getSessionId } from 'calypso/landing/stepper/utils/use-session-id';
+import { Navigate, StepperStep } from '../../types';
```

Comment:
Nit: these should be type imports.
```suggestion
import type { Navigate, StepperStep } from '../../types';
```